app/services/user.js

```diff
@@ -26,3 +28,29 @@ exports.createUser = query => {
     throw errors.defaultError(e.message);
   }
 };
+exports.loginUser = async body => {
+  if (!body.email || !body.password) {
+    throw errors.badRequest('Missing attribute');
+  }
+  try {
+    const result = await User.findAll({
+      where: {
+        email: body.email
+      }
+    });
+    if (result.length === 0) {
+      throw errors.badRequest('User not found');
```

Comment:
bad request is an 400 error. If you respond `not found`, the api rest standar says that the correct error is 404.